menas/src/main/scala/za/co/absa/enceladus/menas/HDFSConfig.scala

```diff
@@ -15,29 +15,24 @@
 
 package za.co.absa.enceladus.menas
 
-import org.springframework.context.annotation.{ Bean, Configuration }
-import org.springframework.beans.factory.annotation.Value
-import org.slf4j.LoggerFactory
-import org.apache.hadoop.conf.{ Configuration => HadoopConfiguration }
-import org.apache.hadoop.fs.{ FileSystem, Path }
-import org.ietf.jgss.Oid
-import javax.security.auth.login.LoginContext
-import javax.security.auth.Subject
-import scala.util.Try
-import org.apache.hadoop.security.UserGroupInformation
-import javax.security.auth.callback._;
-import org.apache.hadoop.security.SecurityUtil
+import org.apache.hadoop.conf.{Configuration => HadoopConfiguration}
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.hadoop.security.{SecurityUtil, UserGroupInformation}
 import org.apache.hadoop.security.UserGroupInformation.AuthenticationMethod
-import org.springframework.beans.factory.annotation.Autowired
 import org.apache.spark.sql.SparkSession
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.annotation.{Autowired, Value}
+import org.springframework.context.annotation.{Bean, Configuration}
 
 @Configuration
 class HDFSConfig @Autowired()(spark: SparkSession) {
   private val logger = LoggerFactory.getLogger(this.getClass)
 
-  @Value("${za.co.absa.enceladus.menas.hadoop.auth.method}") 
+  @Value("${za.co.absa.enceladus.menas.hadoop.auth.method}")
   val authMethod: String = ""
-  @Value("${za.co.absa.enceladus.menas.hadoop.auth.krb5.debug:}") 
+  @Value("${za.co.absa.enceladus.menas.hadoop.auth.user}")
```

Comment:
```suggestion
  @Value("${za.co.absa.enceladus.menas.hadoop.auth.user:}")
```
Shouldn't this be an optional configuration, since this is only used in the "simple" auth case? 
In Spring the expression language (`Value("${...}") allows you to specify that the property may not be set by appending a colon `:` at the end so it won't blow up on start-up.